hbase-backup/src/test/java/org/apache/hadoop/hbase/backup/TestIncrementalBackup.java

```diff
@@ -164,6 +178,7 @@ public void TestIncBackupRestore() throws Exception {
         .setColumnFamily(ColumnFamilyDescriptorBuilder.of(fam2Name)).removeColumnFamily(fam3Name)
         .build();
       TEST_UTIL.getAdmin().modifyTable(newTable1Desc);
+      takeFullBackup(tables, client);
```

Comment:
Related to [this PR](https://github.com/apache/hbase/pull/6340). Happy to wait for that one to be merged before we merge this one in so this makes more sense